docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Why are we using lowercase <code>t</code> here? Lowercase <code>t</code> typically refers to the element inside of a subsequence or inside of the time series <code>T</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
I assume the comment is for the previous version of notebook. So, I ignore this. If I am not correct, please let me know.